package.json

```diff
@@ -8,16 +8,16 @@
     "node": ">=4.0"
   },
   "repository": "googleapis/nodejs-error-reporting",
-  "main": "./src/index.js",
+  "main": "./build/src/index.js",
   "nyc": {
     "exclude": [
-      "./utils/fuzzer.js"
+      "./build/utils/fuzzer.js"
     ]
   },
   "files": [
     "src",
-    "utils",
-    "index.js"
+    "build/utils",
```

Comment:
Should `src` still exist in this list? Why do we ship `utils` as part of the package?

Reply:
I've updated the list to only include `build/src`.  `build/utils` shouldn't be needed since it is only used by the unit and system tests.